python/pyhail/context.py

```diff
@@ -25,24 +43,48 @@ def __init__(self, sc):
 
         self.jsc.hadoopConfiguration().set(
             'io.compression.codecs',
-            'org.apache.hadoop.io.compress.DefaultCodec,org.broadinstitute.hail.io.compress.BGzipCodec,org.apache.hadoop.io.compress.GzipCodec')
+            'org.apache.hadoop.io.compress.DefaultCodec,'
+            'org.broadinstitute.hail.io.compress.BGzipCodec,'
+            'org.apache.hadoop.io.compress.GzipCodec')
+
+        if log is None and not quiet:
+            import os
+            log = os.getcwd() + '/hail.log'
+
+        scala_package_object(self.jvm.org.broadinstitute.hail.driver).configure(
+            self.jsc,
+            log,
+            quiet,
+            append,
+            parquet_compression,
+            block_size,
+            branching_factor,
+            tmp_dir)
 
         logger = sc._jvm.org.apache.log4j
```

Comment:
You can remove this log4j stuff.  I think it is superseded by the Main configuration.